Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:18-slim
+
+WORKDIR /app
+
+COPY . .
+
+RUN yarn install
+
+EXPOSE 8888/tcp
+
+CMD ["yarn", "start"]
```

Comment:
In the current production env we use 
```
/usr/bin/node /opt/apollon/build/server/bundle.js
```
To start the application - Maybe someone with more experience with Apollon could explain the difference? From my understanding `yarn start` is only meant for development. 



I guess we also need some env variables as well - At least we have 
```
NODE_ENV=production
``` 
on the prod server

Reply:
`yarn build` uses webpack to produce a `dist` folder with the static content, so we don't need to run any node application as far as I can tell.
We decided to use that static content in the Dockerfile and add an http server (nginx) resulting in an Image size of just 23MB.